src/marqo/core/unstructured_vespa_index/constants.py

```diff
@@ -0,0 +1,7 @@
+# The threshold for a string to be considered short, long in unstructured Vespa.
+# Users will be able to filter on short strings, but not on long strings.
+SHORT_STRING_THRESHOLD = 20
```

Comment:
You could move constants to common to be consistent with structured index package